Add default behavior to ggshield secret ignore (#960)
GitGuardian/ggshield: changelog.d/20240912_120216_salome.voltz_add_ignore_hash_cmd.md

### Added

> - Default behavior for `ggshield secret ignore` command to allow ignoring secrets using their hash with optional `--name` argument.

Reviewer: "Default behavior" is a bit too implementation oriented.

```suggestion
- It is now possible to ignore a secret manually using `ggshield secret ignore MATCH --name NAME`.
```
